presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -80,7 +80,7 @@ Constructors
 
 .. function:: ST_Point(double, double) -> Point
 
-    Returns a geometry type point object with the given coordinate values.
+    Returns a geometry type point object with the given x and y.  For geospatial functions, the order is longitude then latitude.
```

Comment:
This comment applies to all ST_ functions, hence, it seems inconsistent to make it at a function level. How would you feel about changing (double, double) to (x, y) instead?

Reply:
Do we want to maintain the type information, like `(x: double, y: double)` (or similar).  Or should that also be in the top-level description?

Reply:
It looks like we don't do this anywhere in the docs. I think (x, y) will be fine.